src/components/case_manager/types/table.ts

```diff
@@ -0,0 +1,45 @@
+export interface Person {
+  first_name: string;
+  last_name: string;
+  email: string;
+  phone?: string;
```

Comment:
Remove phone and dateAdded for now. In general, it is easier to add fields/data to interfaces later than to remove, so err on the side of minimalism.

Also remove other non-essential fields in the other interfaces